iroh-bytes/Cargo.toml

```diff
@@ -43,6 +43,7 @@ tracing-futures = "0.2.5"
 http-body = "0.4.5"
 iroh-test = { path = "../iroh-test" }
 proptest = "1.0.0"
+serde_test = "1.0.176"
```

Comment:
is it enough to use just "1"?

Reply:
this is how `cargo add` does it, which I don't think changing has a lot of benefit